src/Elcodi/Component/Page/ElcodiPageTypes.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014 Elcodi.com
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Page;
+
+/**
+ * Class ElcodiPageTypes
+ */
+final class ElcodiPageTypes
+{
+    /**
+     * Page type regular
+     *
+     * Used in footer
+     */
+    const TYPE_REGULAR = 1;
+
+    /**
+     * Page type blog post
+     *
+     * Used in blog
+     */
+    const TYPE_BLOG_POST = 2;
```

Comment:
I'd rather have would go with a string `group` field, so you can request all of the pages related to a group, and create your own groups (also, this would make collecting urls easier). But I think that's something we can do in the beta time.